CONTRIBUTING.md

```diff
@@ -198,6 +198,10 @@ The current Rust plugin modules:
 * `:js` - interop with JavaScript language
 * `:ml-completion` - integration with [Machine Learning Code Completion](https://github.com/JetBrains/intellij-community/tree/master/plugins/completion-ml-ranking) plugin
 
+The current Toml plugin modules:
+* `:intellij-toml` - contains plugin meta-information & icons
```

Comment:
Also, this module is used as a root module to build toml plugin. Let's unify description of this module and `:plugin` one because they used for the same purposes